### DISCUSSION TITLE
How to handle "unknown" or unidentified creators

### DISCUSSION THREAD

**Discussion by sarayourfriend** [WordPress/openverse#3508] [General]
@krysal raised this here: https://github.com/WordPress/openverse/issues/3496#issuecomment-1847668438

In response to @AetherUnbound creating that issue using the "Unknown" Smithsonian creator, Krystle asked:

> This is a more interesting case, actually. I'll assume the creator is not really named "Unknow," but we don't know who the creator is. How is this case being handled? Do we show all the unknown creator's works under that source?

I'm pulling this into a separate discussion because it's unrelated to the issue Madison opened.

**Comment by sarayourfriend** [comment]
"Unknown" is rather unfortunate, and it's a direct result of us not using smithsonian's metadata in a way that leads to the best presentation of metadata. I agree though, @krysal, that for _at least this provider_ it's not a useful creator page and could go unlinked. I can't think of where else to link it that doesn't present its own confusions/difficulties.

The issue is that a museum is not going to name the creator of an _item_ if they don't know it. "Unknown" here probably means that there Smithsonian has not (and presumably has no way to) identified the creator of the object. It is almost certainly not related to the _image_ of the item.

For these cases, I think it's appropriate to leave out the _creator_ and instead credit the institution as the _provider_, if that makes sense. I think you shared examples of how CC expects that to work.

**Reply by AetherUnbound** [reply]
This is an interesting point, I wonder how many other providers it might make sense to make a similar exception for. There may also be providers (Flickr, for example) that have users/creators whose name is literally "Unknown". That seems like it might conflate with our use of "Unknown" here too, although perhaps that's a different issue 😅

**Reply by sarayourfriend** [reply]
I don't think there's a blanket approach for all providers, but it is probably safe to assume that all GLAM institutions following normal cataloguing practices would have something along the lines of "The creator of this object is unknown". It would need to be on a per-provider level. There's no way this makes sense for Flickr, iNaturalist, etc. Except, of course, for GLAM institutions that use Flickr (and Wikimedia) as their distribution. In those cases we're really incorrectly attributing the creator as the source, when it should probably follow the source attribution (particularly if it's PDM) approach that Krystle shared elsewhere.

**Reply by sarayourfriend** [reply]
A key to this though is that there is no single keyword we can apply to all providers to detect this, and we probably need to have flexibility to either catch and indicate it during ingestion (which makes it most flexible, per-provider), or to assign a characteristic to a source (in the ContentProvider object in the API, perhaps) to indicate that its works should not be credited as _creator_ but rather as source ("provided by ... under PDM ..." etc).

Which makes me think there are three baseline features needed:

1. Some way of designating works for which the creator is unknown at ingestion time. For example, if the provider follows a known huristic for identifying these works that we can then apply a boolean flag to the work to mark the work as not having a known creator. I think "creator" might technically be a nullable field. If it is, we could leave it blank and call it a day, but that's harder to tell from a mistake than an intentional flag that says "the creator of this work is not known".
2. A way of designating sources, both those within larger providers like Flickr and Wikimedia, as well as provider-sources like GLAM institutions, that should be credited as providers rather than creators.
3. Implementing this approach to attribution in Openverse.org and the API's attribution code, and then asking Gutenberg and Jetpack to update their implementations, if it isn't directly based on one of ours.

Here's what a I mean by crediting the source as provider: https://wiki.creativecommons.org/wiki/Recommended_practices_for_attribution#Citing_public_domain_materials (this is the page that Krystle linked before, just able to find it again now). Pulling their example:

> “[The Milkmaid](http://hdl.handle.net/10934/RM0001.COLLECT.6417)” by Vermeer. The Rijksmuseum collection. The image is dedicated to the public domain under [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/).

Here Rijksmuseum is credited as the provider of the work with Vermeer as the creator. For GLAM institutions they should be credited as the provider in the attribution text. I've heard in person from library and museum folks alike that this is something they want people to do, and heard one person from a library with a prominent digital collection who was pretty miffed by someone using PDM images from their collection for commercial purposes (totally allowed) without making any attribution to the library. That's just anecdotal to say that LIS/GLAM institutions really do care about this, so it'd be good to do.
